lib/gettext/compiler.ex

```diff
@@ -111,45 +126,100 @@ defmodule Gettext.Compiler do
             __CALLER__,
             __MODULE__,
             domain,
+            msgctxt,
             {msgid, msgid_plural},
             Gettext.Compiler.get_and_flush_extracted_comments()
           )
         end
 
-        {msgid, msgid_plural}
+        {{msgid, msgid_plural}, msgctxt}
```

Comment:
Please make it return a three element tuple instead.

Reply:
I tried that first, but I got an error saying that this is not a correct AST node. My pragmatic approach was then to turn it into this tuple. See the error message below after running `mix test`:
```elixir
== Compilation error in file test/gettext_test.exs ==
** (CompileError) test/gettext_test.exs: invalid quoted expression: {"You have one message, %{name}", "You have %{count} messages, %{name}", nil}

Please make sure your quoted expressions are made of valid AST nodes. If you would like to introduce a value into the AST, such as a four-element tuple or a map, make sure to call Macro.escape/1 before
    expanding macro: GettextTest.Translator.dpngettext_noop/4
    test/gettext_test.exs:486: GettextTest."test dngettext/5"/1
    expanding macro: GettextTest.Translator.dpngettext/6
    test/gettext_test.exs:486: GettextTest."test dngettext/5"/1
    expanding macro: GettextTest.Translator.dngettext/5
    test/gettext_test.exs:486: GettextTest."test dngettext/5"/1
    (ex_unit) expanding macro: ExUnit.Assertions.assert/1
    test/gettext_test.exs:486: GettextTest."test dngettext/5"/1
    (elixir) lib/kernel/parallel_compiler.ex:237: anonymous fn/4 in Kernel.ParallelCompiler.spawn_workers/7
```

Reply:
Make it return this then:

```suggestion
        {:{}, [], [msgid, msgid_plural, msgctxt]}
```

Which is the AST for a  three element tuple. :)

Reply:
If the above doesn't work, then keep it as is and i will take a look later!

Reply:
Will do!